content/influxdb/cloud/organizations/bucket-schema/_index.md

```diff
@@ -0,0 +1,147 @@
+
+   Provide the following:
+   - location of your schema columns file with the `columns-file` flag
+   - measurement name with the `name` flag. This will match the [measurement column](/influxdb/cloud/reference/key-concepts/data-elements/#measurement) in your data.
+
+   To output schema column details, use the optional `extended-output` flag.
+
+   ```sh
+   influx bucket-schema create \
+     --bucket my_explicit_bucket \
+     --name cpu \
+     --columns-file columns.csv \
+     --extended-output
+   ```
+
+   The output is the following:
+
+   ```sh
+   ID                      Measurement Name       Column Name     Column Type   Column Data Type    Bucket ID
+   <schema-id>             cpu                    time            timestamp                        <bucket-id>
+   <schema-id>             cpu                    host            tag                              <bucket-id>
+   <schema-id>             cpu                    usage_user      field         float              <bucket-id>
+   ```
+
+4. To write data to the bucket, use the [`influx write` command](/influxdb/cloud/reference/cli/influx/write).
+
+### Add schemas to a bucket
+Use the [`influx bucket-schema create` command](/influxdb/cloud/reference/cli/influx/bucket-schema/create) to define additional measurement
+schemas for a bucket.
+
+{{% note %}}
+{{< bucket-schema/requires >}}
+```sh
+{{< get-assets-text "bucket-schema/bucket-schema-type.sh" >}}
+```
+{{% /note %}}
+
+1. Add a schema to define the new measurement.
+
+    ```sh
+    influx bucket-schema create \
+      --bucket my_explicit_bucket \
+      --name temperature \
+      --columns-file schema.json
+    ```
+
+2. Use the [`influx bucket-schema list` command](/influxdb/cloud/reference/cli/influx/bucket-schema/list) to list bucket-schemas.
+   To output schema column details, use the optional `extended-output` flag.
+
+    ```sh
+    influx bucket-schema list --bucket my_explicit_bucket --extended-output
+    ```
+
+### Update a schema
+
+  Use the [`influx bucket-schema update` command](/influxdb/cloud/reference/cli/influx/bucket-schema/update) to add new columns to a schema. You cannot modify or delete columns from your schema.
+
+### Errors
+
+#### Not permitted by schema
+  If the measurement data doesn't conform to the bucket schema for that measurement, InfluxDB returns an error.
+
+  The following example attempts to write a *cpu* measurement with an incorrect `usage_user` [data type](/influxdb/cloud/reference/glossary/#data-type):
+
+  ```sh
+  influx write -b my_explicit_bucket 'cpu,host=myHost usage_user="1001" 1556896326'
+  ```
+
+  The output is the following:
+  ```sh
+  Error: failed to write data:
+  unable to parse 'cpu,host=myHost usage_user="1001" 1556896326':
+  schema: field type for field "usage_user" not permitted by schema; got String but expected Float
+  ```
```

Comment:
No need to indent these paragraphs.

~~~suggestion
Use the [`influx bucket-schema update` command](/influxdb/cloud/reference/cli/influx/bucket-schema/update) to add new columns to a schema. You cannot modify or delete columns from your schema.

### Errors

#### Not permitted by schema
If the measurement data doesn't conform to the bucket schema for that measurement, InfluxDB returns an error.

The following example attempts to write a *cpu* measurement with an incorrect `usage_user` [data type](/influxdb/cloud/reference/glossary/#data-type):

```sh
influx write -b my_explicit_bucket 'cpu,host=myHost usage_user="1001" 1556896326'
```

The output is the following:
```sh
Error: failed to write data:
unable to parse 'cpu,host=myHost usage_user="1001" 1556896326':
schema: field type for field "usage_user" not permitted by schema; got String but expected Float
```
~~~